index.js

```diff
@@ -300,10 +424,21 @@ async function run() {
     }
     const taskDefArn = registerResponse.taskDefinition.taskDefinitionArn;
     core.setOutput('task-definition-arn', taskDefArn);
+    
+    // Run the task outside of the service
+    const clusterName = cluster ? cluster : 'default';
+    const shouldRunTaskInput = core.getInput('run-task', { required: false }) || 'false';
+    const shouldRunTask = shouldRunTaskInput.toLowerCase() === 'true';
+    core.debug(`shouldRunTask: ${shouldRunTask}`);
+    
+    if (shouldRunTask) {
+      core.debug("Running one-off task...");
+      await runTask(ecs, capacityProviderStrategy, clusterName, taskDefArn, waitForMinutes);
+    }
 
     // Update the service with the new task definition
     if (service) {
-      const clusterName = cluster ? cluster : 'default';
+      //const clusterName = cluster ? cluster : 'default';
```

Comment:
Is this `//` intentional?

Reply:
This is the original code that assigns the corresponding values to the cluster but adding the one-off task feature runs everything in one cluster. I commented it out for testing reasons, Should i just fully delete it ?